hibernate-core/src/test/java/org/hibernate/orm/test/jpa/callbacks/CallbacksDisabledTest.java

```diff
@@ -39,28 +38,19 @@
 public class CallbacksDisabledTest {
 
 	@Test
-	public void testCallbacksAreDisabled(EntityManagerFactoryScope scope) throws Exception {
-		int id = scope.fromTransaction(
+	public void testCallbacksAreDisabled(EntityManagerFactoryScope scope) {
+		scope.inTransaction(
 				entityManager -> {
 					Cat c = new Cat();
 					c.setName( "Kitty" );
 					c.setDateOfBirth( new Date( 90, 11, 15 ) );
 					entityManager.persist( c );
-					return c.getId();
-				}
-		);
-
-		scope.inTransaction(
-				entityManager -> {
-					Cat _c = entityManager.find( Cat.class, id );
+					entityManager.getTransaction().commit();
+					entityManager.clear();
+					entityManager.getTransaction().begin();
+					Cat _c = entityManager.find( Cat.class, c.getId() );
 					assertTrue( _c.getAge() == 0 ); // With listeners enabled this would be false. Proven by org.hibernate.orm.test.jpa.callbacks.CallbacksTest.testCallbackMethod
 				}
 		);
-
-		scope.inTransaction(
-				entityManager -> {
-					entityManager.createQuery( "delete from Cat" ).executeUpdate();
-				}
-		);
```

Comment:
Did you not suggest that I should review that instances be removed at the end of each test? Ok, I not always use @AfterEach, but I think that is a matter of there being more than one test or not (iirc)